### DISCUSSION TITLE
SQLConnection

### DISCUSSION THREAD

**Discussion by TMNASaW** [microsoft/vscode-discussions#320] [Extension Development QnA] [answered]
Hi.. wondering if we can do SQLConnection and other classes for connecting to SQL server via c# in VSCode?  I am not able to find the SQLConnection class. Tried including the namespace, but find only System.Data.Common and System.Data.SQLTypes which is not helpful.

**Comment by burkeholland** [comment, marked as answer]
Is this a question about an extension? It sounds like this might be a more generic C# question. If so, this is probably better answered via StackOverflow or the C# extension GitHub Issues.

Closing for now in favor of the above options. If you believe this is an error, please feel free to re-open.
